docker-compose/nginx/Dockerfile

```diff
@@ -0,0 +1,155 @@
+FROM alpine:3.9
```

Comment:
Can we explain either in the README or in this same file why we need nginx? Whether is required or optional?